dipper/graph/Graph.py

```diff
@@ -20,12 +20,14 @@ class Graph(metaclass=ABCMeta):
 
     @abstractmethod
     def addTriple(
-            self,
-            subject_id,
-            predicate_id,
-            object_id,
-            object_is_literal=False,
-            literal_type=None):
+            self, 
+            subject_id, 
+            predicate_id, 
+            object_id, 
+            object_is_literal, 
```

Comment:
Shouldn't the sensible defaults for `object_is_literal` and `literal_type` be preserved?

Reply:
agreed, fixed this